### ISSUE TITLE
Test suite fails randomly

### ISSUE THREAD

**Issue by nfm** [peterkeen/payola#161]
It looks like the test suite has at least some tests that are order dependent. Running the suite multiple times at current master (1c5ac87a4262ef8635bd0d51a9a790f04f282daa) returns a variety of different results.

A passing seed is `bundle exec rspec --seed 54985`.

Some failing seeds are `bundle exec rspec --seed 48662` and `bundle exec rspec --seed 9860` (different numbers of errors, and different specs that fail).

I haven't looked into what could be causing this at all yet.

**Comment by peterkeen** [comment]
Thanks for this. I just ran the build on CI a few times and yeah, it looks like it just randomly fails sometimes. I never see it locally, which is super frustrating.

**Comment by nfm** [comment]
@peterkeen Hrm that's kinda weird that it never fails for you locally. Is that the case if you run it with an explicit seed that was failing for me?

There might be some discrepancy between the gems you have and those that I have (or CI has) installed. But I think the RSpec random ordering feature is kind of old though (added in 2.11.0), and I'm not sure what else could be affecting this repro.

Here's my Gemfile.lock:

```ruby
PATH
  remote: .
  specs:
    payola-payments (1.3.2)
      aasm (>= 4.0.7)
      jquery-rails
      rails (>= 4.1)
      stripe (= 1.20.1)
      stripe_event (>= 1.3.0)

GEM
  remote: https://rubygems.org/
  specs:
    aasm (4.3.0)
    actionmailer (4.2.4)
      actionpack (= 4.2.4)
      actionview (= 4.2.4)
      activejob (= 4.2.4)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 1.0, >= 1.0.5)
    actionpack (4.2.4)
      actionview (= 4.2.4)
      activesupport (= 4.2.4)
      rack (~> 1.6)
      rack-test (~> 0.6.2)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (4.2.4)
      activesupport (= 4.2.4)
      builder (~> 3.1)
      erubis (~> 2.7.0)
      rails-dom-testing (~> 1.0, >= 1.0.5)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    activejob (4.2.4)
      activesupport (= 4.2.4)
      globalid (>= 0.3.0)
    activemodel (4.2.4)
      activesupport (= 4.2.4)
      builder (~> 3.1)
    activerecord (4.2.4)
      activemodel (= 4.2.4)
      activesupport (= 4.2.4)
      arel (~> 6.0)
    activesupport (4.2.4)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    arel (6.0.3)
    blankslate (3.1.3)
    builder (3.2.2)
    celluloid (0.15.2)
      timers (~> 1.1.0)
    codeclimate-test-reporter (0.4.8)
      simplecov (>= 0.7.1, < 1.0.0)
    dante (0.2.0)
    diff-lcs (1.2.5)
    docile (1.1.5)
    docverter (1.0.1)
      rest-client (~> 1.7)
    domain_name (0.5.24)
      unf (>= 0.0.5, < 1.0.0)
    erubis (2.7.0)
    factory_girl (4.5.0)
      activesupport (>= 3.0.0)
    factory_girl_rails (4.5.0)
      factory_girl (~> 4.5.0)
      railties (>= 3.0.0)
    globalid (0.3.6)
      activesupport (>= 4.1.0)
    http-cookie (1.0.2)
      domain_name (~> 0.5)
    i18n (0.7.0)
    jimson-temp (0.9.5)
      blankslate (>= 3.1.2)
      multi_json (~> 1.0)
      rack (~> 1.4)
      rest-client (~> 1.0)
    jquery-rails (4.0.5)
      rails-dom-testing (~> 1.0)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    json (1.8.3)
    loofah (2.0.3)
      nokogiri (>= 1.5.9)
    mail (2.6.3)
      mime-types (>= 1.16, < 3)
    mime-types (2.6.2)
    mini_portile (0.6.2)
    minitest (5.8.0)
    multi_json (1.11.2)
    netrc (0.10.3)
    nokogiri (1.6.6.2)
      mini_portile (~> 0.6.0)
    rack (1.6.4)
    rack-test (0.6.3)
      rack (>= 1.0)
    rails (4.2.4)
      actionmailer (= 4.2.4)
      actionpack (= 4.2.4)
      actionview (= 4.2.4)
      activejob (= 4.2.4)
      activemodel (= 4.2.4)
      activerecord (= 4.2.4)
      activesupport (= 4.2.4)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.2.4)
      sprockets-rails
    rails-deprecated_sanitizer (1.0.3)
      activesupport (>= 4.2.0.alpha)
    rails-dom-testing (1.0.7)
      activesupport (>= 4.2.0.beta, < 5.0)
      nokogiri (~> 1.6.0)
      rails-deprecated_sanitizer (>= 1.0.1)
    rails-html-sanitizer (1.0.2)
      loofah (~> 2.0)
    railties (4.2.4)
      actionpack (= 4.2.4)
      activesupport (= 4.2.4)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (10.4.2)
    rest-client (1.8.0)
      http-cookie (>= 1.0.2, < 2.0)
      mime-types (>= 1.16, < 3.0)
      netrc (~> 0.7)
    rspec-core (3.3.2)
      rspec-support (~> 3.3.0)
    rspec-expectations (3.3.1)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.3.0)
    rspec-mocks (3.3.2)
      diff-lcs (>= 1.2.0, < 2.0)
      rspec-support (~> 3.3.0)
    rspec-rails (3.3.3)
      actionpack (>= 3.0, < 4.3)
      activesupport (>= 3.0, < 4.3)
      railties (>= 3.0, < 4.3)
      rspec-core (~> 3.3.0)
      rspec-expectations (~> 3.3.0)
      rspec-mocks (~> 3.3.0)
      rspec-support (~> 3.3.0)
    rspec-support (3.3.0)
    rspec_junit_formatter (0.2.3)
      builder (< 4)
      rspec-core (>= 2, < 4, != 2.12.0)
    simplecov (0.10.0)
      docile (~> 1.1.0)
      json (~> 1.8)
      simplecov-html (~> 0.10.0)
    simplecov-html (0.10.0)
    sprockets (3.3.4)
      rack (~> 1.0)
    sprockets-rails (2.3.3)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      sprockets (>= 2.8, < 4.0)
    sqlite3 (1.3.10)
    stripe (1.20.1)
      json (~> 1.8.1)
      mime-types (>= 1.25, < 3.0)
      rest-client (~> 1.4)
    stripe-ruby-mock (2.1.0)
      dante (>= 0.2.0)
      jimson-temp
      stripe (>= 1.20.1)
    stripe_event (1.5.0)
      activesupport (>= 3.1)
      stripe (~> 1.6)
    sucker_punch (1.2.1)
      celluloid (~> 0.15.2)
    thor (0.19.1)
    thread_safe (0.3.5)
    timers (1.1.0)
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.1)

PLATFORMS
  ruby

DEPENDENCIES
  codeclimate-test-reporter
  docverter
  factory_girl_rails
  payola-payments!
  rspec-rails
  rspec_junit_formatter
  simplecov
  sqlite3
  stripe-ruby-mock (= 2.1.0)
  sucker_punch (~> 1.2.1)

BUNDLED WITH
   1.10.5
```

**Comment by nfm** [comment]
This should be fixed in #164 - I inadvertently introduced ordering bugs when I added the `Payola.create_stripe_plans` config!
